cmd/neofs-cli/modules/acl/extended/create.go

```diff
@@ -74,33 +71,19 @@ func createEACL(cmd *cobra.Command, _ []string) {
 		}
 	}
 
-	rulesFile, err := getRulesFromFile(fileArg)
```

Comment:
Why did you change this call?

Reply:
To make code simpler, thought once there are only one place of call getRulesFromFile() and this function doing trivial thing it is better to remove it.

Reply:
It makes the diff bigger for no reason, can we leave it?

Reply:
Sure, reverted.